### ISSUE TITLE
Question about the train

### ISSUE THREAD

**Issue by clanguagesh** [WU-Wenhao/PseudoSiameseTeacher#3]
In your paper, you state:

“We warm-up the base detector on the annotated subset by 60 epochs followed by training on the mixture of annotated subset and unannotated subset by 8 epochs. We employ SGD as the optimizer with a momentum of 0.9 and weight decay of 0.0001. The initial learning rate decays at 40 and 55 epochs in the warm-up stage and at epoch 6 in the fine-tune stage. The batch size is 2, with one image from the annotated subset and one image from the unannotated subset during the fine-tune stage.”

However, the provided code does not seem to implement this two-stage training strategy. Could you please update the code accordingly?
